appveyor.yml

```diff
@@ -0,0 +1,72 @@
+environment:
+  matrix:
+    - PYTHON: "C:\\Python34"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "32"
+
+platform: x86
+
+configuration: Release
+
+init:
+  - cmd: ECHO Processor architecture - %PROCESSOR_ARCHITECTURE%
+  - cmd: wmic OS get OSArchitecture
+
+  # As AppVeyor has multiple python install, verify which one uses by default
+  - cmd: ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%
+  - cmd: python --version
+  - cmd: python -c "import struct; print(struct.calcsize('P') * 8)"
+
+  # Set the relevant python and pip location to the path
+  - cmd: set PATH=%PYTHON%;%PYTHON%\scripts;%PATH%
+  - cmd: ECHO Path - %PATH%
+
+  # Verify the new default python
+  - cmd: python --version
+  - cmd: python -c "import struct; print(struct.calcsize('P') * 8)"
+  - cmd: pip --version
+
+  # Check out installed python packages
+  - cmd: pip freeze
+
+install:
+  # Temporary fix for PyQt not being installed in correct directory: https://github.com/appveyor/ci/issues/363
+  - REG ADD HKCU\Software\Python\PythonCore\3.4\InstallPath /f /ve /t REG_SZ /d C:\Python34
+
+  # Download PyQt5 with 10min timeout, rename to pyqt5_installer.exe, and install
+  - ps: Start-FileDownload 'http://downloads.sourceforge.net/project/pyqt/PyQt5/PyQt-5.4.2/PyQt5-5.4.2-gpl-Py3.4-Qt5.4.2-x32.exe' -FileName pyqt5_installer.exe -Timeout 600000
+  - cmd: pyqt5_installer.exe /S
+  - cmd: python -c "import PyQt5"
+
+  # Install python dependencies
+  - cmd: pip install pyinstaller
+
+  # Check installed packages again
+  - cmd: pip freeze
+
+  # Build mu using PyInstaller and rename executable with timestamp
+  - cmd: pyinstaller package\pyinstaller.spec
+  - cmd: dir dist
+  - ps: Rename-Item .\dist\mu.exe mu-$(get-date -f yyyy-MM-dd_HH_mm_ss).exe
+
+# Not a project with an msbuild file, build done at install.
+build: None
+
+# No tests for now
+#test_script: None
+
+# Push artifacts to s3 bucket and list all
+before_deploy:
+  - ps: Get-ChildItem .\dist\*.exe | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name -DeploymentName ardublockly-s3-deployment }
+  - ps: foreach($artifactName in $artifacts.keys) { $artifacts[$artifactName] }
+
+# Deploy build to Amazon S3 bucket
+deploy:
+  name: ardublockly-s3-deployment
+  provider: S3
+  access_key_id: AKIAJYJV7NN6HVHCX5NQ
+  secret_access_key:
+    secure: PlLCQKTcf9IzBXpEnXUxbJifb0usS7qcghnM0VxBTX0IL3C975JPidrYjP39ge7P
+  bucket: ardublockly-builds
```

Comment:
I'm not quite sure if there is another way to configure this in appveyor (which doesn't mean there isn't one, but after a recent search I haven't encountered any other way). Looks like [secured variables](http://www.appveyor.com/docs/build-configuration#secure-variables) will always show up in their encrypted form in the yml file.
The project settings do allow to do all this configuration in their website, instead of using a yml file (it also offers the option to export everything into a yml file), but I assumed it was one way or the other. Maybe we can have only the deployment data in the web interface, and everything else executed from the config file, I'll have to look into it.